resources/codegen/tests/resources/openassetio-codegen-test-all.yaml

```diff
@@ -0,0 +1,72 @@
+# yaml-language-server: $schema=../../python/openassetio_codegen/schema.json
+# yamllint disable-line rule:document-start
+package: openassetio-codegen-test-all
+
+description:
+    Test classes to validate the integrity of the openassetio-codegen tool.
+
+traits:
+    namespace:
```

Comment:
[consider][minor] At first glance it's not clear that `namespace` is a namespace, i.e. it could be declaring that what comes next is a namespace, perhaps `aNamespace` or `someNamespace` or similar. Super minor though.

Reply:
How's [this](https://github.com/OpenAssetIO/OpenAssetIO/pull/529/commits/b24c317b511fa1926866a8f24f5aa3909e7eab1d).

Reply:
Yeah much better, sorry about the quantity of changes required though!

Reply:
No - it was a good call!